blueprint/duktape/src-noline/duk_config.h

```diff
@@ -2898,7 +2898,7 @@ typedef struct duk_hthread duk_context;
 #undef DUK_USE_DEBUGGER_DUMPHEAP
 #undef DUK_USE_DEBUGGER_INSPECT
 #undef DUK_USE_DEBUGGER_PAUSE_UNCAUGHT
-#undef DUK_USE_DEBUGGER_SUPPORT
+#define DUK_USE_DEBUGGER_SUPPORT
```

Comment:
Yea, definitely approaching the time where we should figure out the right way to do this :)

My only thought here– this would enable for Release mode too yea? I feel like it's probably helpful to compile Release without the debugger support

Reply:
Yeah indeed. I'm changing a couple more of these defines so I'll workout how to switch this between debug and release sensibly. We're happy just hacking this file inline for now though then?

Reply:
:+1: as long as we can do it sensibly between debug/release, yea I think we can make this change inline.

When this commit goes in, let's add a new issue to discuss how to better integrate duktape and referring to this commit to clarify/track that we edited these files inline. That way we keep track of these changes as we consider better Duktape integrations

Reply:
OK Awesome. I'll get this all cleaned up.

Reply:
Will move all the new defines to bottom of duk_config.h with a comment and wrap under JUCE_DEBUG.